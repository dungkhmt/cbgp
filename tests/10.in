20 20
20 36
20 11
11 2
2 3
3 9
9 12
12 8
8 20
5 13
13 4
4 16
16 15
15 7
7 5
6 19
19 1
1 14
14 18
18 6
18 7
6 1
17 10
15 9
10 14
12 15
16 9
13 19
5 8
20 2
5 6
17 14
16 3
12 7
10 18
4 2
20 13
4 1
1
0 0 20 2 21 3 22 0 11 3 8 2 18 10 16 6 22 4 14 14 17 2 18 8 13 1 12 12 19 9 28 0 14 13 17 11 9 1 18 4 
