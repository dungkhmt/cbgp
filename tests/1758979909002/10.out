(19, 10), (10, 1), (1, 2), (2, 8), (8, 11), (11, 7), (7, 19), (4, 12), (12, 3), (3, 15), (15, 14), (14, 6), (6, 4), (5, 18), (18, 0), (0, 13), (13, 17), (17, 5), (17, 6), (5, 0), (16, 9), (14, 8), (9, 13), (11, 14), (15, 8), (12, 18), (4, 7), (19, 1), (4, 5), (16, 13), (15, 2), (11, 6), (9, 17), (3, 1), (19, 12), (3, 0), 
numberIntersection=0 
minAngle=0.49216336723716925 
Var(edgeLength)=-47.81136901597382 
minDistance=2.1213203435596424 
Var(nodePos)=-276.8275 
sumAngle=17.96338357718141 
sumDistance=12321.591006213717 
minEdgeLength=4.47213595499958 
sumEdgeLength=368.5564583902875 

0.0000000000 0.4921633672 -47.8113690160 2.1213203436 -276.8275000000 17.9633835772 12321.5910062137 4.4721359550 368.5564583903

0: (5, 2),
1: (34, 7),
2: (39, 12),
3: (32, 0),
4: (24, 17),
5: (15, 15),
6: (30, 26),
7: (31, 17),
8: (40, 18),
9: (18, 44),
10: (29, 6),
11: (36, 22),
12: (24, 8),
13: (11, 40),
14: (42, 25),
15: (45, 13),
16: (16, 40),
17: (19, 31),
18: (14, 6),
19: (29, 11),

Time taken: 32.507 s
5 2 34 7 39 12 32 0 24 17 15 15 30 26 31 17 40 18 18 44 29 6 36 22 24 8 11 40 42 25 45 13 16 40 19 31 14 6 29 11 
0: (5, 2), 1: (34, 7), 2: (39, 12), 3: (32, 0), 4: (24, 17), 5: (15, 15), 6: (30, 26), 7: (31, 17), 8: (40, 18), 9: (18, 44), 10: (29, 6), 11: (36, 22), 12: (24, 8), 13: (11, 40), 14: (42, 25), 15: (45, 13), 16: (16, 40), 17: (19, 31), 18: (14, 6), 19: (29, 11), 
numberIntersection=0 minAngle=0.49216336723716925 sumAngle=17.96338357719037 minEdgeLength=4.47213595499958 sumEdgeLength=368.5564583902878 minDistance=2.1213203435596424 sumDistance=12321.591006213712 
